packages/itwin/grouping-mapping-widget/src/widget/components/customUI/GroupingMappingCustomUI.ts

```diff
@@ -95,9 +95,9 @@ export interface GroupingCustomUIProps {
   /**
    * To get the query loading status.
    */
-  isUpdating?: boolean;
+  isUpdating: boolean;
```

Comment:
Why did you introduce this breaking change?

Reply:
Reverted